bugsnag-plugin-android-ndk/src/main/jni/event.h

```diff
@@ -109,6 +109,7 @@ typedef struct {
     char network_access[64];
     char os_build[64];
     char os_version[64];
+    char os_name[64];
```

Comment:
Also missing free disk and free memory

Reply:
It's not possible to capture these values in a C signal handler so we omit them from the `device` struct.